block-storage/src/test/scala/coop/rchain/blockstorage/BlockStoreTest.scala

```diff
@@ -38,8 +38,19 @@ trait BlockStoreTest
     PropertyCheckConfiguration(minSuccessful = PosInt(100))
 
   private[this] def toBlockMessage(bh: BlockHash, v: Long, ts: Long): BlockMessage =
-    BlockMessage(blockHash = bh)
-      .withHeader(Header().withVersion(v).withTimestamp(ts))
+    BlockMessage
+      .from(
+        BlockMessageProto()
+          .withBlockHash(bh)
+          .withHeader(
+            HeaderProto()
+              .withVersion(v)
+              .withTimestamp(ts)
+          )
+          .withBody(BodyProto().withState(RChainStateProto()))
+      )
+      .right
+      .get
```

Comment:
It could be part of CasperMessageFactory too

Reply:
I've moved `CasperMessageFactory` to `models/test` and renamed it to `Dummies`

Reply:
My bigger point here was to use the Dummies.blockMessage in here and other places in tests that create default-valued ADTs via protos for test setup. Won't push on that though, this can be done in the future :)